overlord/devicestate/firstboot.go

```diff
@@ -314,10 +357,20 @@ func (m *DeviceManager) populateStateFromSeedImpl(tm timings.Measurer) ([]*state
 	ts := tsAll[len(tsAll)-1]
 	endTs := state.NewTaskSet()
 
+	// Start tracking any validation sets included in the seed after
+	// installing the included snaps.
+	if !preseed {
```

Comment:
as we discussed we don't want this afaik

Reply:
Removed this